src/Digdir.Tool.Dialogporten.SlackNotifier/Digdir.Tool.Dialogporten.SlackNotifier.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
Bør endres til isolated worker process, in-process-modellen dør med .net6, se https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide

Reply:
Nydelig tilbakemelding! Kode ble 100% bedre med isolated worker process!